### PR TITLE
Modal alert will be displayed if the user tries to start a new game #43

### PR DESCRIPTION
**Pull request by nazgul7d**
hey guys,
I wrote a function for Modal alert
Please let me know if you have any comments

### CODE REVIEW

**Review by goodman2814 — APPROVED** [review]
looks good to me!

**Review by PaulaR-05 — CHANGES_REQUESTED** [review]
Naz! Great job!

I really like your code and also the way you built the logic behind it.
What do you think? 

thanks a lot for all your work :)

My only suggestion is to change that part of the code:
**const occupiedSquares = document.querySelectorAll(".occupied");** 
to
**const occupiedSquares = document.querySelectorAll(".block");**



Because i believe it would be more intresting if the user had submited his answer on the grid.

File: app.js [hunk 1]
```diff
@@ -131,14 +131,31 @@ submitButton.addEventListener("click", function () {
...
 
 // Add event listener to new game button
-
 newGameButton.addEventListener("click", function () {
-  clearGrid();
-  rollThatDice();
-  emptyLeaderboard();
-  window.location.reload();
+  // Check if any squares are marked as occupied
+  const occupiedSquares = document.querySelectorAll(".occupied");
```
Comment (PaulaR-05) [inline app.js:136]: const occupiedSquares = document.querySelectorAll(".block");

it's the only part I would change, the function is really great :)

**Review by PaulaR-05 — APPROVED** [review]
Thanks a lot, Naz! Great Job

**Pull request merged**